app/src/main/java/com/uni/gruppenphaseandroid/InGameFragment.java

```diff
@@ -72,5 +72,11 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             NavHostFragment.findNavController(InGameFragment.this)
                     .navigate(R.id.action_InGameFragment_to_FirstFragment);
         });
+
+
+        view.findViewById(R.id.fab_cardholder).setOnClickListener(view1 -> {
```

Comment:
Wofür das hier? Das schaut für mich so aus als wolltest du zu einem neuen Fragment wechseln wenn man den cardholder anklickt.

Reply:
genau... ich habe den "Cardholder" in ein Fragment gepackt, damit ich da dann auch die Sensor Auswertung machen kann, fürs schummeln
und Cardholer ist im Grunde nur ein Botton, der wie das Chat Symbol die ganze zeit in der ecke zugänglich ist
ist es klug so und die beste Lösung? keine Ahnung